Extensions/blacklist.css

```diff
@@ -211,3 +210,7 @@
 	position: relative;
 	top: 0;
 }
+
+.xblacklist_excuse ~ * {
+	display: none;
+j}
```

Comment:
<https://www.youtube.com/watch?v=DNm5eb97Y0g>

Reply:
Vim is a burden